data/json/items/comestibles.json

```diff
@@ -13617,6 +13617,44 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "willowbark",
+        "name" : "willow bark",
+        "weight" : 323,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You eat some willow bark.",
+            "effects" : [
+                {
+                    "id": "pkill_l",
```

Comment:
It shouldn't really have any painkiller effect on its own; until it's synthesized with an acetyl group, it's just plain salicylic acid - which isn't much of a painkiller, and more of a toxin.